R/scale.R

```diff
@@ -110,6 +110,20 @@ prep.step_scale <- function(x, training, info = NULL, ...) {
 
   sds <- sds * x$factor
 
+  which_sd <- which(sds ==0)
+  if (length(which_sd) > 0) {
+    with_na <- names(which_sd)
+    with_na_str <- paste(paste0("`", with_na, "`"), collapse = ", ")
+    rlang::abort(c(
```

Comment:
Should this be `rlang::warn()`? I think probably yes.

Reply:
Ok I can change it back, I will also change 'x' to 'i' since it doesn't stop R anymore

Reply:
Done